pkg/stages/steps_init.go

```diff
@@ -353,27 +353,6 @@ func GetCleanupStage(sis values.System, l types.KairosLogger) []schema.Stage {
 		},
 	}
 
-	var pkgs []values.VersionMap
-	if !config.DefaultConfig.TrustedBoot {
-		// This packages are used to build the initramfs and are not needed anymore
-		pkgs = append(pkgs, values.ImmucorePackages[sis.Distro][values.ArchCommon])
-		pkgs = append(pkgs, values.ImmucorePackages[sis.Family][values.ArchCommon])
-		pkgs = append(pkgs, values.ImmucorePackages[sis.Distro][sis.Arch])
-		pkgs = append(pkgs, values.ImmucorePackages[sis.Family][sis.Arch])
-	}
-
-	filteredPkgs := values.FilterPackagesOnConstraint(sis, l, pkgs)
```

Comment:
remove this function too? Is it needed elsewhere?

Reply:
yes, on package selection for package install